src/Filter/GetFilter.php

```diff
@@ -18,6 +17,6 @@ public function filter(string $property, ?object $instance = null): bool
             $pos = 0;
         }
 
-        return substr($property, $pos, 3) === 'get';
+        return strpos($property, 'get', $pos) === $pos;
```

Comment:
I'd use `str_starts_with()` vs `strpos()` in the two separate execution branches, tbh